inductiva/simulators/dualsphysics.py

```diff
@@ -17,21 +17,22 @@ def run(
         self,
         input_dir: types.Path,
         commands: List[dict],
-        machine_group: Optional[resources.MachineGroup] = None,
+        on: Optional[resources.MachineGroup] = None,
         storage_dir: Optional[types.Path] = "",
     ) -> tasks.Task:
         """Executes a DualSPHysics simulation.
 
         Args:
             input_dir: Directory with simulation input files.
             sim_config_filename: Simulation config file.
-            machine_group: Machine group for simulation.
+            on: The computational resource to launch the simulation in. If None
+                the simulation is launched in a machine of the default pool.
```

Comment:
Make `in` `on` the same either on the method argument or on the documentation

```suggestion
            on: The computational resource to launch the simulation on. If None
                the simulation is launched in a machine of the default pool.
```